docs/step-by-step/step-10.md

```diff
@@ -149,6 +149,16 @@ the following:
 sudo certbot --nginx
 ```
 
+If you receive an `error` after executing the command, like the one shown below: 
+`Saving debug log to /var/log/letsencrypt/letsencrypt.log`
+`The requested nginx plugin does not appear to be installed`
+
+Use this command:
+```bash
+sudo apt-get install python-certbot-nginx
+```
+
+
```

Comment:
Hey! Nice catch, and thanks for taking the time to investigate this and send in a PR. Let's put this into a blockquote, since it's more of a secondary notice that (hopefully) only certain users will encounter. The syntax is a little convoluted, given every line needs to start with a `>`, but let me know if you have questions implementing it.

````suggestion
> See this error after running `sudo certbot --nginx`?
>
> ```
> Saving debug log to /var/log/letsencrypt/letsencrypt.log
> The requested nginx plugin does not appear to be installed`
> ```
>
> You must install `python-certbox-nginx`. On Ubuntu or Debian systems, you can run `sudo apt-get install
> python-certbot-nginx` to download and install this package.
````

Reply:
Hi, thank you for fixing the code styling. No clue what to place to make it look good